docs/en/integrations/language-clients/python/index.md

```diff
@@ -0,0 +1,316 @@
+---
+sidebar_label: ClickHouse Connect
+sidebar_position: 1
+keywords: [clickhouse, python, client, connect, integrate]
+slug: /en/integrations/python
+description: A suite of Python packages for connecting Python to ClickHouse
+---
+
+# ClickHouse Connect
```

Comment:
Could you add a link to this file as done in https://github.com/ClickHouse/clickhouse-docs/pull/412/?